WordPress/Classes/Utility/CoreDataStoreMover.swift

```diff
@@ -0,0 +1,64 @@
+import Foundation
+
+class CoreDataStoreMover {
+    enum MoveError: Error {
+        case destinationFileExists(url: URL)
+        case sourceFileDoesNotExist(url: URL)
+        case couldNotLoadMetadataForStore(url: URL, error: Error)
+        case couldNotLoadModel
+        case couldNotLoadSourceStore(url: URL, error: Error)
+        case sourceStoreLoadedButNotAvailable(url: URL)
+        case couldNotBackupDatabase(error: Error)
+        case couldNotMigrateStore(error: Error)
+    }
+
+    /// Moves the Store from `sourceLocation` to `targetLocation`.
+    ///
+    /// - Returns: on success this method returns a boolean holding `true` if the store was moved, or `false` if it wasn't
```

Comment:
It seems that on success, an url is returned, not a bool. Maybe the comment needs to be updated?

Reply:
Updated.